src/session.rs

```diff
@@ -322,11 +322,11 @@ impl Client {
     }
 
     /// Issue the specified [`WebDriverCompatibleCommand`] to the WebDriver instance.
-    pub async fn issue_cmd(
-        &self,
-        cmd: Box<dyn WebDriverCompatibleCommand + Send>,
-    ) -> Result<Json, error::CmdError> {
-        self.issue(Cmd::WebDriver(cmd)).await
+    pub async fn issue_cmd<C>(&self, cmd: C) -> Result<Json, error::CmdError>
+    where
+        C: WebDriverCompatibleCommand + Send + 'static,
```

Comment:
Thinking some more about this, I _think_ we may want
```rust
C: Into<Box<dyn WebDriverCompatibleCommand + Send + 'static>>
```
so that if someone passes in a `Box<dyn WebDriverCompatibleCommand>`, we don't end up double-boxing it. I _think_ that should work. You _may_ need to write (if it's even legal):
```rust
impl From<Box<T>> for Box<dyn WebDriverCompatibleCommand + Send + 'static> where T: WebDriverCompatibleCommand + Send + 'static {
}
```

Reply:
So if I do 
```rust
impl<T> From<Box<T>> ...
```

it works for Box<T> but not T.

The only way I found around it was to do 
```rust
impl<T> From<T> for Box<...>
```
but now we're back to double-boxing 🤷

Reply:
I've simplified it for now - still boxing internally though.
I've added tests for this as well.